src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import style from './style.scss';
+
+const cx = classNames.bind(style);
+
+class Progressbar extends React.Component {
+  constructor (props) {
+    super(props);
+    const { percentage, showPercentageNum, className } = props;
+
+    this.percentage = percentage || 0;
+    this.showPercentageNum = showPercentageNum;
+    this.className = className;
+  }
+
+  getBarClass () {
+    return cx(
+      'progress-bar',
+      this.className);
+  }
+
+  render () {
+    const progressBarWidth = this.percentage > 100 ? 100 : this.percentage;
+    const percentageLabel = this.showPercentageNum ? `${this.percentage}%` : null;
+
+    return (
+      <div>
+        <div className={cx('progress-bar-container')}>
+          <div style={{ width: `${progressBarWidth}%` }} className={this.getBarClass()}>
+            {percentageLabel}
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+Progressbar.propTypes = {
+  className: PropTypes.string,
+  percentage: PropTypes.number,
+  showPercentageNum: PropTypes.bool,
```

Comment:
I guess `percentage` is mandatory data, then it should be required.

Reply:
What about to have option for flat or rounded? for example, `isFlat` is true, then the progressbar is rectangle, if it is false, then it has rounded corner.

Reply:
Also, please take a look at other components to get some idea how to documentation for this props. Having some brief information for each props would be helpful for general users.

Reply:
sure.